### PR TITLE
feat: Add artisan db:table command

### PR DESCRIPTION
**Pull request by almas1992**
## 📑 Description

This PR introduces an artisan `db:table` command to display table information.

#### Screenshots
image

image

@coderabbitai summary

## ✅ Checks

- [x] Added test cases for my code

### CODE REVIEW

**Comment by almas1992** [comment]
Review Ready

File: database/console/table_command_test.go [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+				mockSchema.EXPECT().GetIndexes("test").Return(nil, nil)
+				mockSchema.EXPECT().GetForeignKeys("test").Return(nil, assert.AnError)
+				mockContext.EXPECT().Error(mock.Anything).Run(func(message string) {
+					color.Errorln(message)
+				})
+			},
+			expected: assert.AnError.Error(),
+		},
+		{
+			name: "success",
```
Comment (hwbrzzl) [inline database/console/table_command_test.go:128]: No expected in this case? We can use Contains to assert it if it's a bit complicated.

File: database/console/table_command_test.go [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+	}
+	tests := []struct {
+		name     string
+		setup    func()
+		expected string
+	}{
+		{
+			name: "get tables failed",
+			setup: func() {
+				mockContext.EXPECT().NewLine()
```
Comment (hwbrzzl) [inline database/console/table_command_test.go:36]: Ditto

```suggestion
				mockContext.EXPECT().NewLine().Once()
```

File: database/console/table_command_test.go [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+				})
+			},
+			expected: assert.AnError.Error(),
+		},
+		{
+			name: "get foreign keys failed",
+			setup: func() {
+				mockContext.EXPECT().NewLine()
+				mockContext.EXPECT().Option("database").Return("")
+				mockSchema.EXPECT().Connection(mock.Anything).Return(mockSchema)
```
Comment (hwbrzzl) [inline database/console/table_command_test.go:114]: Avoid using `mock.Anything` if we can.

**Review by hwbrzzl — COMMENTED** [review]
Amazing, it's easier since we have Schema.

**Comment by almas1992** [comment]
Review Ready

File: database/console/table_command.go [hunk 4]
```diff
@@ -39,8 +39,9 @@ func (r *TableCommand) Extend() command.Extend {
 		ArgsUsage: "  [--] [<table>]",
 		Flags: []command.Flag{
 			&command.StringFlag{
-				Name:  "database",
-				Usage: "The database connection",
+				Name:    "connection",
```
Comment (hwbrzzl) [inline database/console/table_command.go:42]: The same with `db:show`, we can keep `database` since Laravel uses `database`.

**Comment by almas1992** [comment]
Review Ready

**Review by hwbrzzl — APPROVED** [review]
Perfect

**Pull request merged**